stdlib/builtins.pyi

```diff
@@ -335,7 +335,7 @@ class _FormatMapMapping(Protocol):
 
 class str(Sequence[str]):
     @overload
-    def __new__(cls: Type[_T], o: object = ...) -> _T: ...
+    def __new__(cls: Type[_T], object: object = ...) -> _T: ...
```

Comment:
This one *is* a positional-or-keyword argument! But it has the wrong argument name, currently.